docs/source/_static/style.css

```diff
@@ -1,3 +1,12 @@
 .wy-nav-content {
     max-width: 1000px !important;
 }
+/* make links red and bold */
+code.xref {
```

Comment:
This makes cross-references (which I _think_ are only things like :class:, :meth:, etc) red instead of black